programmers/난이도별/level02.주식가격/dkdlelk99.py

```diff
@@ -0,0 +1,20 @@
+def solution(prices):
+    answer = []
+    len_prices = len(prices)
+    for i in range(len_prices):
+        up_price = 0
+        for j in range(i + 1, len_prices):
+            up_price += 1
+            if prices[i] > prices[j]:
+                answer.append(up_price)
+                break
+            if j == len_prices - 1:
+                answer.append(up_price)
+        if i == len_prices - 1:
+            answer.append(0)
```

Comment:
```suggestion
        for j in range(i + 1, len_prices):
            up_price += 1
            if prices[i] > prices[j]:
                break
        answer.append(up_price)
```

@RockMiin 님 코드를 참고하여 셀프리뷰했습니다. 깔끔하게 푸셨네요!

Reply:
감사합니다 ㅎㅎ 저랑 비슷하게 푸신 것 같네요 ! 고생하셨습니다 :)